source/blog/2017-02-16-hyperconverged-ovirt-with-ovn.md

```diff
@@ -0,0 +1,355 @@
+At this point you have to create your first storage domain for regular VMs on the second gluster volume (the hosted-engine storage domain is a special one and could contain just the engine VM).
+So go to the storage tab and choose new domain; set Domain Function *Data*, Storage Type *GlusterFS*, set *Use managed gluster* and choose the existing *data* volume. Set *Data* as storage domain name and enter *backup-volfile-servers=192.168.1.202:192.168.1.203* (this ensure that everything should work also if the first host is down without any single point of failure) in the mount option field as for picture 14.
+![2017-02-16-hyperconverged-ovirt-with-ovn-picture14.png](2017-02-16-hyperconverged-ovirt-with-ovn-picture14.png "2017-02-16-hyperconverged-ovirt-with-ovn-picture14.png")
+Once the first regular storage domain goes up also the datacenter should go up and the hosted-engine storage domain and the engine VM should be automatically imported by the engine; wait for them to apper as for picture 15.
+![2017-02-16-hyperconverged-ovirt-with-ovn-picture15.png](2017-02-16-hyperconverged-ovirt-with-ovn-picture15.png "2017-02-16-hyperconverged-ovirt-with-ovn-picture15.png")
+Once they are in you should add the two additional hosts but, before that, since we skipped the DNS infrastructure, we should also two entries for the two additional hosts under /etc/hosts on the engine VM:
+```
+cat <<EOF >> /etc/hosts
+192.168.1.202 demovm2.localdomain demovm2
+192.168.1.203 demovm3.localdomain demovm3
+EOF
+```
+At this point we can go to the hosts tab and choose new host setting name=address=*demovm2.localdomain* and its root password in the general tab and *deploy* in the hosted-engine tab as for pictures 16 and 17.
+![2017-02-16-hyperconverged-ovirt-with-ovn-picture16.png](2017-02-16-hyperconverged-ovirt-with-ovn-picture16.png "2017-02-16-hyperconverged-ovirt-with-ovn-picture16.png")
+![2017-02-16-hyperconverged-ovirt-with-ovn-picture17.png](2017-02-16-hyperconverged-ovirt-with-ovn-picture17.png "2017-02-16-hyperconverged-ovirt-with-ovn-picture17.png")
+Repeat it for the third host and wait for both of them to come up as for picture 18. The hosts are ready for hosted-engine when they are shown with a little crown: the host where the engine VM is running is shown with a gold crown, the other two with silver crown.
+![2017-02-16-hyperconverged-ovirt-with-ovn-picture18.png](2017-02-16-hyperconverged-ovirt-with-ovn-picture18.png "2017-02-16-hyperconverged-ovirt-with-ovn-picture18.png")
+The running status of hosted-engine could be checked also from cockpit running on any of the participants hosts as for picture 19.
+![2017-02-16-hyperconverged-ovirt-with-ovn-picture19.png](2017-02-16-hyperconverged-ovirt-with-ovn-picture19.png "2017-02-16-hyperconverged-ovirt-with-ovn-picture19.png")
+The last important step is to check the HA configuration of each host:
+```
+[root@demovm3 ~]# cat /etc/ovirt-hosted-engine/hosted-engine.conf 
```

Comment:
It's not required, it's just to show that oivrt-ha-agent could mount the volume from any of the three hosts.